Doc/library/sched.rst

```diff
@@ -101,6 +103,9 @@ Scheduler Objects
    Remove the event from the queue. If *event* is not an event currently in the
    queue, this method will raise a :exc:`ValueError`.
 
+.. method:: schduler.cancel_all()
+
+   Removes all events from the queue. Causes run() to return immediately.
```

Comment:
Reading the other parts of the file, I think you need to use the infinitive mood. So "**Removes** ..." becomes "**Remove** ..." and I think the same thing should be applied for "**Causes**" as well.

Reply:
Good point!